backend/src/app.service.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@nestjs/common';
+import { Neo4jService } from 'nest-neo4j/dist';
+import { Node } from './entities/Node';
+import { EdgeDescriptor } from './entities/EdgeDescriptor';
+import { Edge } from './entities/Edge';
+import { QueryResult } from './entities/queries/QueryResult';
+import { LimitQuery } from './entities/queries/LimitQuery';
+import { NodeDescriptor } from './entities/NodeDescriptor';
+
+@Injectable()
+export class AppService {
+  constructor(private readonly neo4jService: Neo4jService) {}
+
+  async queryAll(query?: LimitQuery): Promise<QueryResult> {
+    return {
+      edges: query?.limit?.edges === 0 ? [] : await this.getAllEdges(query),
+      nodes: query?.limit?.nodes === 0 ? [] : await this.getAllNodes(query),
+    };
+  }
+
+  /**
+   * Returns a list the ids of all nodes
```

Comment:
This comment seems to be out of date.
Maybe also describe the purpose of the `query` parameter.